packages/evergreen-badges/src/components/Badge.js

```diff
@@ -0,0 +1,58 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Strong } from 'evergreen-typography'
+import colors from 'evergreen-colors'
+import BadgeAppearances from '../styles/badge-appearances'
+
+const getBadgeStyle = ({ appearance, isSolid }) => {
+  const colorGroup = colors[appearance]
+  if (isSolid) {
+    return {
+      backgroundColor: colorGroup['500'],
+      color: 'white',
+    }
+  }
+  return {
+    backgroundColor: colorGroup['15A'],
+    color: colorGroup['1000'],
+  }
+}
+
+export default class Badge extends PureComponent {
+  static propTypes = {
+    ...Text.propTypes,
+    appearance: PropTypes.oneOf(Object.keys(BadgeAppearances)).isRequired,
+  }
+
+  static defaultProps = {
+    appearance: 'neutral',
+    paddingTop: 0,
+    paddingBottom: 0,
+    display: 'inline-block',
+    width: 80,
```

Comment:
Width should not be set, the fact the pills are 80 in the design is just because Sketch needs symbols to be the same dimension to be able to switch them out.